[2025-03-14T23:20:02.536+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T23:20:02.625+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T23:15:00+00:00 [queued]>
[2025-03-14T23:20:02.629+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T23:15:00+00:00 [queued]>
[2025-03-14T23:20:02.629+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T23:20:02.634+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-14 23:15:00+00:00
[2025-03-14T23:20:02.637+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3560) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T23:20:02.638+0000] {standard_task_runner.py:72} INFO - Started process 3561 to run task
[2025-03-14T23:20:02.639+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'scheduled__2025-03-14T23:15:00+00:00', '--job-id', '282', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmpou4s2s0j']
[2025-03-14T23:20:02.639+0000] {standard_task_runner.py:105} INFO - Job 282: Subtask transform
[2025-03-14T23:20:02.658+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform scheduled__2025-03-14T23:15:00+00:00 [running]> on host c20e6545b232
[2025-03-14T23:20:02.702+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T23:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T23:15:00+00:00'
[2025-03-14T23:20:02.703+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T23:20:02.708+0000] {open_weather_dag.py:38} INFO - Starting transformation...
[2025-03-14T23:20:02.709+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-14T23:20:02.709+0000] {open_weather_dag.py:42} INFO - Transformation complete.
[2025-03-14T23:20:02.709+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T23:20:02.714+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T23:20:02.715+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=scheduled__2025-03-14T23:15:00+00:00, execution_date=20250314T231500, start_date=20250314T232002, end_date=20250314T232002
[2025-03-14T23:20:02.738+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T23:20:02.748+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T23:20:02.748+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
